main/introduction.bs

```diff
@@ -18,6 +18,12 @@ with mechanisms for controlling who can access what.
 A <dfn export>Solid app</dfn> is an application
 that reads or writes data from one or more [=data pods=].
 
+A <dfn export>read operation</dfn> entails that information about a resource's existence or its description can be known. [[Source](https://github.com/solid/specification/issues/149#issue-568433265)]
```

Comment:
Is the intent for these citations to github issues / comments to persist indefinitely, or only during the drafting process?

Reply:
Only drafting so that there is a record somewhere (commit history) so that it could be look up if needed. I'm open to the idea of having them persist after.

Reply:
Happy to have them in there during drafting so that addresses my short-term question. Agree it'd be good to keep track of them long-term, though I think we should discuss _where_ to do so when we get to that point.